docs/modules/hive/pages/usage-guide/database-driver-example.adoc

```diff
@@ -0,0 +1,176 @@
+
+= Database drivers
+
+The Stackable Apache Hive product images come with built-in support for using Postgresql as the metastore  database. To use another supported database it is necessary to make the relevant drivers available to Hive: this tutorial shows how this is done for MySql.
```

Comment:
Official names use capitalized SQL: PostgreSQL, MySQL

Reply:
Thanks for spotting that! Done.